eisenhower-matrix/src/main/webapp/script.js

```diff
@@ -41,6 +41,20 @@ function loginStatus(status) {
   }
 }
 
+// checks if task if empty or only includes whitespace
+function checkTitle () {
+    let title = document.getElementById("taskName");
+    title.value = title.value.replace(/^\s+/, '').replace(/\s+$/, '');
+
+    if (title.value === '') {
+        /* title is empty or only whitespace */
+        document.getElementById('taskName').style.borderColor = "red";
```

Comment:
this doesn't prevent them from submitting though, does it? You can check out this tutorial to make sure they won't be able to ignore this red border: [https://www.yourhtmlsource.com/javascript/formvalidation.html](https://www.yourhtmlsource.com/javascript/formvalidation.html)

Reply:
Before the changes, entering only whitespace allowed the task to be added to the page. The task details were posted but the title was empty. 
Thanks for the resource!

Reply:
ok thanks for explaining :)